### DISCUSSION TITLE
Budibase API Calls to Locally Hosted Install

### DISCUSSION THREAD

**Discussion by crsnchng** [Budibase/budibase#5409] [Help] [answered]
Hello,

I've got Budibase running on a Linux VM that is accessible via a public IP. After reviewing #1915 and messing around with both the official Postman and my own Postman collection, I cannot seem to get API calls to the local installation working. I have not received anything other than a 500 or 400. I have also tried curl via the terminal and get the same results. 

I've changed API keys and I've changed what ports are open to the VM. 

My calls are structured like this: http://{public ip}}:10000/api/tables/tableid for example

I've published the app, used the app id without the app_dev_ part, and changed the table's permissions to public. 

As of right now, no API calls seem to be working. Could I be missing something? I have the most recent docker-compose.yaml

Thanks in advance.

**Comment by melohagan** [comment, marked as answer]
Hi @fwit-carson 

It looks like you're trying to use the internal api. 
You'll need to use the public one, which takes the form: `https://{{domain}}/api/public/v1/tables/{{tableId}}`

Docs here: https://docs.budibase.com/reference/get_tables-tableid

**Reply by crsnchng** [reply]
Ah, got it. My calls are working now. Was initially getting errors still but after Publishing the app and using app_{{appId}, everything is working.

Thank you for the reply!
